samples/containers/poetry/src/main/java/com/squareup/sample/poetry/PoemWorkflow.kt

```diff
@@ -107,7 +106,7 @@ object PoemWorkflow : StatefulWorkflow<Poem, Int, ClosePoem, OverviewDetailScree
     sink.writeInt(state)
   }
 
-  private sealed class Action : WorkflowAction<Poem, Int, ClosePoem> {
+  private sealed class Action : MutatorWorkflowAction<Poem, Int, ClosePoem> {
```

Comment:
Well this is just embarrassing, didn't realize I still had stale samples. And such a core one, too.

Reply:
There's a comment further down in the code, this sample was intentionally left using the old API.